spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -1,69 +1,108 @@
 require "rails_helper"
 
 RSpec.describe Broadcasts::WelcomeNotification::Generator, type: :service do
+  let(:mascot_account)             { create(:user) }
+  let!(:welcome_thread)            { create(:article, user: mascot_account, published: true, tags: "welcome") }
+  let!(:welcome_broadcast)         { create(:welcome_broadcast, :active) }
+  let!(:twitter_connect_broadcast) { create(:twitter_connect_broadcast, active: true) }
```

Comment:
```suggestion
  let!(:twitter_connect_broadcast) { create(:twitter_connect_broadcast, :active) }
```
does this work here? wondering why we do it differently between lines 6 and 7 here

Reply:
Oh, I see there's a `trait` in the factory that feels a bit redundant. I'm going to update it.